evmrpc/block.go

```diff
@@ -160,11 +160,12 @@ func EncodeTmBlock(
 	resultHash := common.HexToHash(block.Block.LastResultsHash.String())
 	miner := common.HexToAddress(block.Block.ProposerAddress.String())
 	baseFeePerGas := k.GetBaseFeePerGas(ctx).TruncateInt().BigInt()
-	gasLimit, gasWanted := int64(0), int64(0)
+	var gasWanted int64
```

Comment:
should we name this to gasUsed since it is the sum of `txRes.GasUsed` and returned under the "gasUsed" field?